ui/src/views/PageNotFoundView.vue

```diff
@@ -6,12 +6,19 @@
   >
     <v-row justify="center">
       <v-col cols="auto">
-        page not found
+        <h1>Page not found</h1>
+        <p>The page either doesn't exist or you don't have access to this page</p>
+        <center>
+          <v-img :src="mainPicture" :width="imageWidth" contain />
+        </center>
```

Comment:
I think width doesnt have to be bound to the data here, instead we can use string. And for me personally,  I think its better to use max-height attribute rather than width. Please check the vuetify doc on v-img for more info